tests/admin_views/tests.py

```diff
@@ -4662,6 +4662,17 @@ def test_readonly_get(self):
         response = self.client.get(reverse('admin:admin_views_post_change', args=(p.pk,)))
         self.assertContains(response, "%d amount of cool" % p.pk)
 
+    @ignore_warnings(category=RemovedInDjango20Warning)  # for allow_tags deprecation
+    def test_readonly_text_field(self):
+        p = Post.objects.create(title="Readonly test", content="test",
```

Comment:
Please use 4 space hanging indent style as seen elsewhere in this file such as `cls.superuser = ...`